web/db.py

```diff
@@ -656,7 +656,7 @@ def __init__(self, db_module, keywords):
         self.supports_multiple_insert = False
 
         try:
-            import DBUtils  # noqa, flake8 F401
+            import dbutils  # noqa, flake8 F401
```

Comment:
Please use the proper flake8 ignore syntax
https://flake8.pycqa.org/en/3.9.2/user/violations.html?highlight=noqa#in-line-ignoring-errors